Translated/Story/Unorganized/msc_027020.csv

```diff
@@ -0,0 +1,90 @@
+msc_027020_0010#0,"""Hey, there you are."""
+msc_027020_0020#0,"""What's the situation?"""
+msc_027020_0030#0,"""Did something happen with Faria?"""
+msc_027020_0040#0,"""All of a sudden, Faria started behaving differently<br>right as we were in the middle of talking."""
+msc_027020_0050#0,"""She wanted us to call you here."""
+msc_027020_0060#0,"""All of a sudden, Faria started behaving differently<br>right as we were in the middle of talking."""
+msc_027020_0070#0,"""She wanted us to call you here."""
+msc_027020_0080#0,"""You're all here, then."""
+msc_027020_0090#0,"""... I am Zephetto. I am temporarily borrowing this body."""
+msc_027020_0100#0,"""Faria? What did you just say?"""
+msc_027020_0110#0,"""Zephetto?!"""
+msc_027020_0120#0,"""...Just listen to what I have to say first."""
+msc_027020_0130#0,"""I am currently in a certain location, unable to move."""
+msc_027020_0140#0,"""...Alas, it's unlikely that I can<br>get myself out of this situation on my own."""
+msc_027020_0150#0,"""Which is why I'd like to ask for your help with my escape."""
+msc_027020_0160#0,"""Please come to the coordinates that I'm about to send you."""
+msc_027020_0170#0,"""Wait, just wait a second here."""
+msc_027020_0180#0,"""Are you... really Zephetto?"""
+msc_027020_0190#0,"""I am."""
+msc_027020_0200#0,"""...What happened after the fight in Leciel?"""
+msc_027020_0210#0,"""And what do you mean by \""unable to move\""?"""
+msc_027020_0220#0,"""...And lastly... What's happening to Faria right now?"""
+msc_027020_0230#0,"""I am using an ability to gain control of a portion of Faria's body."""
+msc_027020_0240#0,"""...An ability?"""
+msc_027020_0250#0,"""Faria's own consciousness is unaffected. You've been listening to everything, right, Faria?"""
+msc_027020_0260#0,"""...Yes. I can hear you, Zephetto."""
+msc_027020_0270#0,"""Oh, thank goodness. She's okay, then."""
+msc_027020_0280#0,"""I will explain everything to you at this location."""
+msc_027020_0290#0,"""I've had Faria \""memorize\"" the path up to this point."""
+msc_027020_0300#0,"""Manon, Faria, Aina... And you, 909."""
+msc_027020_0310#0,"""All four of you come here alone."""
+msc_027020_0320#0,"""...I'll be waiting."""
+msc_027020_0330#0,"""...Zephetto?"""
+msc_027020_0340#0,"""..."""
+msc_027020_0350#0,"""Zephetto's consciousness has vanished from within me."""
+msc_027020_0360#0,"""That's the second time he just left after a one-sided conversation..."""
```

Comment:
What about something like "he just infodumped on us and left"? Too modern?

Reply:
way too modern unfortunately lol

Reply:
reworded